tests/rule_based_profiler/data_assistant/test_column_value_missing_data_assistant.py

```diff
@@ -87,3 +90,45 @@ def test_column_value_missing_data_assistant_plot_expectations_and_metrics_corre
     # This test passes only if absense of any metrics and expectations to plot does not cause exceptions to be raised.
     column_domain_charts: List[dict] = [p.to_dict() for p in plot_result.charts[2:]]
     assert len(column_domain_charts) == 0
+
+
+@pytest.mark.integration
+@pytest.mark.slow
```

Comment:
Is this slow?

Reply:
I have seen tests marked slow if they take more than 1 second. What is the threshold for marking slow?

Reply:
it is 1s ... and I think we have a tool that can annotate slow tests with a comment indicating the number of seconds